server/task/task-memory/src/test/java/org/apache/james/task/eventsourcing/MemoryTerminationSubscriberTest.java

```diff
@@ -20,8 +20,18 @@
 
 package org.apache.james.task.eventsourcing;
 
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
 class MemoryTerminationSubscriberTest implements TerminationSubscriberContract {
     public TerminationSubscriber subscriber() {
         return new MemoryTerminationSubscriber();
     }
+
+    @Disabled("JAMES-2813 This test is unstable")
+    @Override
+    @Test
+    public void dynamicListeningEventsShouldGetOnlyNewEvents() {
```

Comment:
is it failing only on the CI? I did never see it, and I cannot reproduce any failure on it after 1000+ runs.

Reply:
Yes it is. Often.

See https://github.com/linagora/james-project/pull/3580#issuecomment-659199301 https://github.com/linagora/james-project/pull/3577#issuecomment-659197378

Reply:
ok I can reproduce it by lowering the delays to 5ms instead of 50ms. It must be due to the CI running on a VM and so having some strange timer behaviors